prow/prowjobs/GoogleCloudPlatform/gcp-guest/compute-image-tools.yaml

```diff
@@ -1,3 +1,32 @@
+postsubmits:
```

Comment:
Switched to postsubmit since I found that running as a presubmit took too long (~30 minutes)